taskcluster/mozillavpn_taskgraph/transforms/release_index.py

```diff
@@ -0,0 +1,20 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from taskgraph.transforms.base import TransformSequence
+transforms = TransformSequence()
+
+
+@transforms.add
+def add_release_index(config, tasks):
+   for task in tasks:
+        if "add-release-index" in task:
+            release_index = task.pop("add-release-index")
+            if release_index and int(config.params["level"]) == 3:
+                name = task['name'].split("/")[0]
+                version = config.params["app_version"]
+                route = f"index.mozillavpn.v2.mozillavpn.releases.{version}.{name}"
```

Comment:
So given these aren't actually releases, I think we should use the following namespace:
`mozillavpn.v2.mozilla-vpn-client.branch.{branch}.{name}`

This lines up well with index route conventions across other projects, and we also already index Decision tasks there:
https://firefox-ci-tc.services.mozilla.com/tasks/index/mozillavpn.v2.mozilla-vpn-client.branch.main